packages/integrations/mdx/test/remark-imgattr.test.js

```diff
@@ -0,0 +1,66 @@
+import { expect } from 'chai';
+import * as cheerio from 'cheerio';
+import { Writable } from 'node:stream';
+
+import { Logger } from '../../../astro/dist/core/logger/core.js';
+import { loadFixture } from '../../../astro/test/test-utils.js';
+
+const FIXTURE_ROOT = new URL('./fixtures/image-remark-imgattr/', import.meta.url);
+
+describe('Testing remark plugins for image processing', () => {
+	/** @type {import('./test-utils').Fixture} */
+	let fixture;
+
+	describe('start dev server', () => {
+		/** @type {import('./test-utils').DevServer} */
+		let devServer;
+		/** @type {Array<{ type: any, level: 'error', message: string; }>} */
+		let logs = [];
+
+		before(async () => {
+			fixture = await loadFixture({
+				root: FIXTURE_ROOT,
+			});
+
+			devServer = await fixture.startDevServer({
+				logger: new Logger({
+					level: 'error',
+					dest: new Writable({
+						objectMode: true,
+						write(event, _, callback) {
+							logs.push(event);
+							callback();
+						},
+					}),
+				}),
+			});
```

Comment:
Since you're not using the logs, you don't need all that part (and the definition on line 18)